examples/basic/main.tf

```diff
@@ -0,0 +1,87 @@
+#####################################################
+# PVS Configuration
+# Copyright 2022 IBM
+#####################################################
+
+# There are discrepencies between the region inputs on the powervs terraform resource, and the vpc ("is") resources
+provider "ibm" {
+  alias            = "ibm-pvs"
+  region           = lookup(local.ibm_pvs_zone_region_map, var.pvs_zone, null)
+  zone             = var.pvs_zone
+  ibmcloud_api_key = var.ibmcloud_api_key != null ? var.ibmcloud_api_key : null
+}
+
+provider "ibm" {
+  alias            = "ibm-is"
+  region           = lookup(local.ibm_pvs_zone_cloud_region_map, var.pvs_zone, null)
+  zone             = var.pvs_zone
+  ibmcloud_api_key = var.ibmcloud_api_key != null ? var.ibmcloud_api_key : null
+}
+
+locals {
+  ibm_pvs_zone_region_map = {
+    "syd04"    = "syd"
+    "syd05"    = "syd"
+    "eu-de-1"  = "eu-de"
+    "eu-de-2"  = "eu-de"
+    "lon04"    = "lon"
+    "lon06"    = "lon"
+    "tok04"    = "tok"
+    "us-east"  = "us-east"
+    "us-south" = "us-south"
+    "dal12"    = "us-south"
+    "tor01"    = "tor"
+    "osa21"    = "osa"
+    "sao01"    = "sao"
+  }
+
+  ibm_pvs_zone_cloud_region_map = {
+    "syd04"    = "au-syd"
+    "syd05"    = "au-syd"
+    "eu-de-1"  = "eu-de"
+    "eu-de-2"  = "eu-de"
+    "lon04"    = "eu-gb"
+    "lon06"    = "eu-gb"
+    "tok04"    = "jp-tok"
+    "us-east"  = "us-east"
+    "us-south" = "us-south"
+    "dal12"    = "us-south"
+    "tor01"    = "ca-tor"
+    "osa21"    = "jp-osa"
+    "sao01"    = "br-sao"
+  }
+}
+
+resource "tls_private_key" "tls_key" {
```

Comment:
Could we just clearly document that this examples is not for production and the reasons - eg: the security notice at https://registry.terraform.io/providers/hashicorp/tls/latest/docs/resources/private_key